app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb

```diff
@@ -3,28 +3,26 @@ class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::IbmCloudCredent
     {
       :component  => 'password-field',
       :label      => N_('IBM Cloud API Key'),
-      :helperText => N_('The API key for IBM Cloud. A value for this field is required if classic user name and classic API key are not provided. A valid connection must have value for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IBM Cloud Classic Infrastructure API Key.'),
+      :helperText => N_('The API key for IBM Cloud. This credential requires either an IBM Cloud API Key or both a Classic Infrastructure User Name and Classic Infrastructure API key.'),
```

Comment:
Does something like this work?

```suggestion
      :helperText => N_('The API key for IBM Cloud. This credential requires either an IBM Cloud API Key or a Classic Infrastructure User Name / API key pair.'),
```

Reply:
Or perhaps

Does something like this work?

```suggestion
      :helperText => N_('The API key for IBM Cloud. This credential requires either an IBM Cloud API Key or a Classic Infrastructure User Name and API key.'),
```

Reply:
yeah, English vs. logical makes that not obvious.  I think I like the second one.  I'm not a fan of duplicating the same thing 3 times here so do wish we could put this logic into the validation for this form though so we can help the user if they don't provide the right things.

Reply:
Agreed - maybe some tabs?  like we do for multiple auths?

Reply:
actually, separate credential types feels easier

Reply:
I agree, let me split off this into a new PR and get the other change in.  I'll create a new credential type and play with that.

Reply:
Ok, split it off into a separate class.  This will allow us to test both IBM Cloud and IBM Classic Infrastructure: https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/23